http/oidc/src/main/java/org/wildfly/security/http/oidc/TokenValidator.java

```diff
@@ -263,7 +263,7 @@ public ErrorCodeValidator.Error validate(JwtContext jwtContext) throws Malformed
                 String atHash = jwtClaims.getStringClaimValue(AT_HASH);
                 String actualHash;
                 try {
-                    actualHash = getAccessTokenHash(accessTokenString, jwsAlgorithm);
+                    actualHash =  getAccessTokenHash(accessTokenString, jwsAlgorithm);
```

Comment:
Just a very small thing, but just noticed that an extra white space was added here. In general, it's good practice to avoid making unrelated white space changes in PRs.